sounds/tests.py

```diff
@@ -239,6 +240,7 @@ def test_sound_delete(self):
         sound = sounds[0]
         sound.change_processing_state("OK")
         sound.change_moderation_state("OK")
+        c = Comment.objects.create(user=user, sound=sound)
```

Comment:
I guess we want to check somewhere that the comment no longer exists too. What about using `Sound.add_comment` to add this comment?